server/database-couch/seed.js

```diff
@@ -16,7 +16,7 @@ const seed = async (start, number) => {
     // batchSize determines how many users get created simultaneously
     // Use batchSize to not overload image service with too many requests at once
     // Batches of 100+ requests known to cause socket error
-    let batchSize = 100;
+    let batchSize = 1000;
```

Comment:
Good idea to create batches!

Reply:
Yeah, without patching I was getting that JS heap out of memory error.